lib/pipeline/pipeline.js

```diff
@@ -4,14 +4,15 @@ const ProcessLauncher = require('../process/processLauncher');
 const utils = require('../utils/utils.js');
 const constants = require('../constants');
 
-require("babel-preset-react");
-require("babel-preset-es2015");
-require("babel-preset-es2016");
-require("babel-preset-es2017");
+// require("babel-preset-react");
+// require("babel-preset-es2015");
+// require("babel-preset-es2016");
+// require("babel-preset-es2017");
```

Comment:
you can remove if it is not useful

Reply:
Good catch. I did remove those lines at one point; then I was changing and trying some different things and forgot to take them back out.

Reply:
Lines removed.